### ISSUE TITLE
Innaccessible shortcuts on non-QWERTY keyboards

### ISSUE THREAD

**Issue by tleb** [excalidraw/excalidraw#3059]
Hi,

First try on the app, I search for a grid. I find it in the shortcuts section and try to toggle it using `Ctrl+'` but nothing. I am using an AZERTY keyboard and I'm not sure why. I've looked at `keydown` & `keyup` events and they are what I expect:

``` javascript
// in the console, in the excalidraw tab
document.addEventListener('keydown', console.log)
document.addEventListener('keyup', console.log)

// the events when I press Ctrl then press ' then release ' then release Ctrl
keydown Control { target: body, key: "Control", charCode: 0, keyCode: 17 }
keydown Control { target: body, key: "'", charCode: 0, keyCode: 52 }
keyup Control { target: body, key: "'", charCode: 0, keyCode: 52 }
keyup { target: body, key: "Control", charCode: 0, keyCode: 17 }
```

**Comment by h7y** [comment]
Apparently backtick is [located somewhere](https://superuser.com/a/254077) else on AZERTY keyboard.

**Comment by tleb** [comment]
Oh, it's a backtick. It's unclear from the help menu. To write a backtick using an AZERTY keyboard, it's `AltGr+7`. The grid still doesn't show up when `Ctrl+AltGr+7` is pressed.

**Comment by dwelle** [comment]
Hello. It's not a backtick, but `'` as you first suggested. We've had problems with this particular key, so we're using a physical location instead of the printable character (in fact, you wouldn't be able to type `Ctrl + '` on AZERTY as that's `Ctrl + 4` which is reserved by the browser).

As such, you *can* press it, but on AZERTY it's the `ù` character (thus `Ctrl + ù`) ↓

![image](https://user-images.githubusercontent.com/5153846/108366726-fae3e700-71f8-11eb-9b14-48c354404ed3.png)

**Comment by ppaspp** [comment]
It would be great if this hint could be shown in the help screen if a language was chosen, which is often used with a non-QWERTY keyboard (for example german or french).

**Comment by dwelle** [comment]
@ppaspp in chrome-based browsers there are things we could do (and we will at some point). Not so much in other browsers.

**Comment by eproxus** [comment]
I'm using Dvorak on macOS and can't for the life of me figure out what key to actually press to show the grid. Can we get a GUI button for this at least? Or re-bindable keys?

**Comment by onemenzel** [comment]
I use a a German keyboard on macOS and I also do have difficulties accessing the keyboard. Most of the shortcuts are in fact inaccessible to anyone with a non-US-keyboard. Here's an example of what I have to use for which feature, despite the help dialog telling me otherwise:

- zoom in: `cmd` + `ß`
- stats for nerds: `option` + `-`
- zen mode: `option` + `y`
- send backward: `cmd` + `ü`
- bring forwards: `cmd` + `+`

… and the list goes on. I'd be grateful for a fix.

**Comment by zsviczian** [comment]
I use a Hungarian keyboard. All the shortcuts are accessible, the only trick is to know where the key would be on an English QWRTY keyboard. BTW, the location of the shortcuts are ergonomically good and follow standards in other apps, thus I am happy the English key locations apply on my keyboard instead of the actual characters... 

Maybe we need logic to display the shortcut hint based on the users keyboard, not the English keyboard keys, or to show a graphic of a keyboard and display the location of the shortcut instead of naming the character.

**Comment by dwelle** [comment]
> BTW, the location of the shortcuts are ergonomically good and follow standards in other apps, thus I am happy the English key locations apply on my keyboard instead of the actual characters...

An interesting point, since ultimate goal was to use the actual layout/language-specific keys (when & where possible). 🤔

**Comment by septatrix** [comment]
It would be really great if keybindings were configurable. Personally I would very much switch the <kbd>Ctrl</kbd> and <kbd>Alt</kbd> keys the majority of programs use Ctrl for drag and copy instead of alt

**Comment by eproxus** [comment]
Still can't enable the grid. Really wish there was a UI button for it at least...

**Comment by maielo** [comment]
> Still can't enable the grid. Really wish there was a UI button for it at least...
> 
> Edit: Oh, and copy-paste doesn't work either 😞

You can use mouse `right click` meanwhile:
![image](https://user-images.githubusercontent.com/8983870/201059086-c6c8fd3d-4f88-43e1-ab50-036bd006f6d5.png)

**Comment by eproxus** [comment]
@maielo Brillant, thanks! (I had an extension blocking right-click modifications enabled, which is why I didn't find it in the first place... my bad)

**Comment by lolouthefox** [comment]
I can't even select some options using an AZERTY keyboard... ALT+Z is ALT+W, that I found. But I cannot select the 4 first items of the toolbar using the keyboard

**Comment by Aqueuse** [comment]
Yes. a menu on the left hamburger who would open a popup with the actual keyboard shortcuts would be great. It would solve this issue by showing the position on the keyboard instead of the assumed letter. Later, it would allow to dev the feature for remapping :  By clicking on one shortcut, you could reassign it.
